src/tools/std20/views/test/reverse.test.cpp

```diff
@@ -0,0 +1,150 @@
+// include Catch2
+#include <catch2/catch_test_macros.hpp>
+
+// what we are testing
+#include "tools/std20/views.hpp"
+
+// other includes
+#include <forward_list>
+#include <list>
+#include <vector>
+#include "tools/std20/algorithm.hpp"
+
+// convenience typedefs
+//using namespace njoy::tools;
+namespace std20 = nano;
+
+SCENARIO( "subrange" ) {
+
+  const std::vector< int > equal = { 2, 1, 0, -1, -2 };
+  const std::vector< int > modified = { 2, 2, 2, 2, 2 };
+
+  GIVEN( "a container with bidirectional iterators" ) {
+
+    std::list< int > values = { -2, -1, 0, 1, 2 };
+
+    WHEN( "when iterators are used" ) {
+
+      auto chunk = values | std20::views::reverse;
+      using Range = decltype(chunk);
+      using Iterator = nano::iterator_t< Range >;
+
+      THEN( "the subrange satisfies the required concepts" ) {
```

Comment:
"subrange" -> "reverse" 

Likewise for other THENs in scenario

Reply:
same